dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ProcessDefinitionServiceImpl.java

```diff
@@ -279,7 +279,7 @@ public Map<String, Object> createProcessDefinition(User loginUser,
         return createDagDefine(loginUser, taskRelationList, processDefinition, taskDefinitionLogs, otherParamsJson);
     }
 
-    private Map<String, Object> createDagDefine(User loginUser,
+    public Map<String, Object> createDagDefine(User loginUser,
```

Comment:
Do we need to add `createDagDefine` at `ProcessDefinitionService` interface?

Reply:
If we consider rewriting `createdagdefine` , can add.